M2/Macaulay2/packages/Macaulay2Doc/functions/package-doc.m2

```diff
@@ -383,6 +383,19 @@ Node
     loadPackage
     needsPackage
 
+Node
+  Key
+    Keywords
+  Headline
+    the list of keywords of a package
+  Description
+    Text
+      A symbol used as the name of an optional argument.  When used
+      with @TO newPackage@, it should provide a list of strings giving
+      the keywords of the package.  All packages should have at least
+      one keyword.  The keywords currently in use correspond to the
```

Comment:
None of the other inputs of `newPackage` have independent documentation, e.g. Authors doesn't have one, and I don't think this one should either. You can add this to the documentation for `newPackage` instead.

Also, the default value for keyword is `{"Uncategorized"}`, so the sentence "All packages should have at least one keyword" is pointless. This is really a higher requirement for distributed packages, which should be documented somewhere else (e.g. in a `CONTRIBUTING.md` file)

Reply:
Fair enough.  It is documented in the [pull requests](https://github.com/Macaulay2/M2/wiki/Pull-requests) wiki page, which will be linked to the [PR template](https://github.com/Macaulay2/M2/pull/3279) once that's been merged into `master`.  So at least that's a start.

I'll go ahead and close this.